x-pack/platform/plugins/private/reporting/server/lib/tasks/run_scheduled_report.ts

```diff
@@ -32,6 +34,17 @@ export class RunScheduledReportTask extends RunReportTask<ScheduledReportTaskPar
     return SCHEDULED_REPORTING_EXECUTE_TYPE;
   }
 
+  protected getQueueTimeout() {
```

Comment:
We want to set the task timeout longer to account for the possible retries, so we use the number of retries as a multiplier